deploy/charts/rook-ceph/values.yaml

```diff
@@ -54,6 +54,9 @@ pspEnable: true
 # Set the priority class for the rook operator deployment if desired
 # priorityClassName: class
 
+# If true, loop devices are allowed to be used for osd
```

Comment:
This is still a feature for a test cluster, correct? I just want to be clear we don't think it should be used in production. Same comment for everywhere else this is copied.
```suggestion
# If true, loop devices are allowed to be used for osds in test clusters
```

Reply:
I fixed it.

Reply:
I also changed the commit and PR title to say that this feature is for test purposes only.